packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.tsx

```diff
@@ -1245,6 +1247,39 @@ class ViewportPositioner extends Foundation<
                 return ViewportPositionerVerticalPositionLabel.undefined;
         }
     };
+
+    /**
+     * Gets the uninitialized state
+     */
+    private getInitialState = (): ViewportPositionerState => {
+        return {
+            disabled: true,
```

Comment:
should this check for props first since disabled is settable?

Reply:
State does not necessarily match props.  Before initialization the component state is disabled and it only gets enabled if not disabled by props *and* gets valid viewport/anchor values.  Setting disabled = false on the prop does not mean the disabled state will be false if the other essential resources needed by the component fail to validate.

Reply:
What if I want to initialize it as open?

Reply:
I'm going to be away next week, pinging @chrisdholt for an opinion.

Reply:
If the prop is set to true then that should override the state appropriately, the state should end up as true and the menu should be visible.

Reply:
@scomea is this an issue where "disabled" means something different than it typically does? There isn't a prop to set disabled as `true`, correct? This is more a function of the internals of the component?

Reply:
Sort of.  There is a disabled prop which disables positioning logic and causes the positioner to just show up where it would in normal document flow.  The internal disabled "state" starts as true and unless it is disabled in props only gets set to false once the component's initialization completes successfully (ie. we get valid anchor & viewport).  If initialization fails to get resources it needs it ultimately ends up in a disabled state which renders the component in normal document flow.  The component remains hidden from view during initialization so we don't get flicker as we determine resources and calculate the position.